R/modify_pd.R

```diff
@@ -0,0 +1,53 @@
+#' Update indention information of parse data
+#'
+#' @param pd A nested or flat parse table that is already enhanced with
+#'   line break and space information via [create_filler()] or
+#'   [create_filler_nested()].
+#' @param indent_by How many spaces should be added after the token of interest.
+#' @name update_indention
+NULL
+
+#' @rdname update_indention
+indent_round <- function(pd, indent_by) {
+  if (any(pd$token == "')'")) {
```

Comment:
I don't think so because we have some tibbles with only one row. Looking at the last token is probably also not a good idea because we have two cases
`mycall(1, 2)` and ` (1 + x)`
Which have their opening brace at the second and the first position respectively. I think that's really the reason why I implemented the approach with `which` in the first place: To get the start right.
I think this function should really handle both cases and it's easier to implement it in one function instead of two. Hence, I propose to go back to the `which` approach, at least for the start. I think the end is unambiguous.